### ISSUE TITLE
No accessible name - "Support the center today" button

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#73]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess.
